static/js/info_overlay.js

```diff
@@ -79,4 +82,17 @@ exports.maybe_show_keyboard_shortcuts = function () {
     exports.show("keyboard-shortcuts");
 };
 
+exports.render_markdown = () => {
+    let unrendered_text;
+    let obj;
+    $.each($(".apply_markdown"), (id, element) => {
+        unrendered_text = element.textContent;
+        obj = {
+            raw_content: unrendered_text,
+        };
+        markdown.apply_markdown(obj);
+        $(element).next().append(obj.content);
```

Comment:
Let's use a more direct selector for this than `.next()` -- we don't want to make deep assumptions about how the HTML is laid out.

Reply:
I think it would be better to append a `td` tag as you suggested below. That will also remove empty td tags.